src/sentry/static/sentry/app/views/organizationGroupDetails/groupEvents.tsx

```diff
@@ -1,8 +1,9 @@
 import React from 'react';
 import {browserHistory} from 'react-router';
+import {RouteComponentProps} from 'react-router/lib/Router';
```

Comment:
This is just in `react-router` as well. We should prob find replace this in the future